libraries/AP_ESC_Telem/AP_ESC_Telem_Backend.h

```diff
@@ -3,7 +3,7 @@
 #include <AP_HAL/AP_HAL.h>
 
 #ifndef HAL_WITH_ESC_TELEM
-#define HAL_WITH_ESC_TELEM HAL_SUPPORT_RCOUT_SERIAL || HAL_MAX_CAN_PROTOCOL_DRIVERS
+#define HAL_WITH_ESC_TELEM 0
```

Comment:
this doesn't look right, off by default?

Reply:
Like the commit message says this commit is just to exercise CI to test the changes. It should not be merged!